tests/suites/ck/ck.sh

```diff
@@ -15,7 +15,7 @@ run_deploy_ck() {
 		sudo snap install kubectl --classic --channel latest/stable
 	fi
 
-	wait_for "active" '.applications["kubernetes-control-plane"] | ."application-status".current'
+	wait_for "active" '.applications["kubernetes-control-plane"] | ."application-status".current' 360
```

Comment:
It's a bit unfortunate that the `wait_for` argument is "max attempts" rather than "max seconds". This looks like 6 minutes (360 seconds) as is, as these sorts of arguments are usually in seconds (cf `sleep 360`). Seeing you just added this recently, how hard would it be to change it to max seconds rather than max attempts?

Reply:
Done